src/molecules/side-menu.tsx

```diff
@@ -22,23 +23,24 @@ const SideMenu = ({ isOpen, closeMenu, subjects }: SideMenuProps) => {
             ×
           </button>
         </div>
+
         <button className={`${styles.menuItemButton} on-background-text`}>
           <div className={styles.stateLayer}>ホーム</div>
         </button>
+
         <div className={styles.subjectList}>
           {subjects &&
-            subjects.map((subject, index) => (
-              <button
-                key={index}
-                className={`${styles.menuItemButton} secondary-container on-secondary-container-text`}
-              >
-                <div className={styles.stateLayer}>
-                  <div className={styles.subjectName}>{subject.name}</div>
-                  <div className={styles.lastDate}>{subject.lastDate}</div>
-                </div>
-              </button>
-            ))}
+            subjects.map((subject) => (
+              <Link key={subject.id} href={`/attendances/${subject.id}`}className={styles.menuItemLink} onClick={closeMenu}>
```

Comment:
```suggestion
              <Link key={subject.id} href={`/attendances?subject_id=${subject.id}`}className={styles.menuItemLink} onClick={closeMenu}>
```
クエリパラメータで制御するようになったので, リンクのパスはこのようになると思います.